dart/lib/src/open_location_code.dart

```diff
@@ -34,7 +34,7 @@ const codeAlphabet = '23456789CFGHJMPQRVWX';
 const encodingBase = codeAlphabet.length;
```

Comment:
_This is an automated bot comment from the TravisCI tests_  
**File has `dartfmt` errors that must be fixed**. Here is a diff, or run `format_check.sh`: